### ISSUE TITLE
Cannot find module './tencentcloud'

### ISSUE THREAD

**Issue by luka26** [serverless-components/tencent-website#10]
when i run  sls git:(master) ✗ sudo sls --debug:
````
  DEBUG ─ Resolving the template's static variables.
  DEBUG ─ Collecting components from the template.
  DEBUG ─ Downloading any NPM components found in the template.
  DEBUG ─ Analyzing the template's components dependencies.
  DEBUG ─ Creating the template's components graph.
  DEBUG ─ Syncing template state.
  DEBUG ─ Executing the template's components graph.

  error:
  { Error: Cannot find module './tencentcloud'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/coding/.serverless/components/registry/npm/@serverless/tencent-website@1.2.19/node_modules/tencentcloud-sdk-nodejs/index.js:1:84)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3) code: 'MODULE_NOT_FOUND' }

  3s › Template › Error: Cannot find module **'./tencentcloud'`**
```
my yml:
```
sls:
  component: '@serverless/tencent-website'
  inputs:
    code:
      src: ./code
      index: index.html
      error: index.html
    region: ap-guangzhou
    bucketName:my-bucket
```

**Comment by tinafangkunding** [comment]
Thanks for reporting! @yugasun can you please check this problem?

**Comment by yugasun** [comment]
Because `tencentcloud-sdk-nodejs@3.0.120` is a bug version, and it had been fix by `tencentcloud-sdk-nodejs@3.0.121`
